.github/workflows/release.yml

```diff
@@ -2,85 +2,108 @@ name: Release
 
 on:
   workflow_dispatch:
```

Comment:
I wonder if manual invocation should still be allowed.

Reply:
I've left it in for now for cases where we might still want to run this manually (eg. if the deployment failed and we want to add a commit for fixing the deployment). But if it turns out we never need it, we can get rid of it later! :)

Reply:
Good point!